CHANGES.md

```diff
@@ -36,6 +36,11 @@ To be released.
         Instead, a newly added static class `StoreExtension` provides
         an extension method of the same name.
      -  Added `IStore.IterateStateReferences()` method.
+ -  `Swarm` constructor became to receive `BlockChain<T>`.  [[#324]]
+ -  `Swarm` became to have type parameter `T` that reprsents action type as
+    like as `BlockChain<T>`.  [[#324]]
+ -  Methods that take `BlockChain<T>` as a parameter in `Swarm` no longer take
+    `BlockChain<T>`. And they are no longer the generic methods.  [[#324]]
```

Comment:
```suggestion
    became to neither longer take `BlockChain<T>` nor a generic method.
    Because the `Swarm` constructor takes it instead.  [[#324]]
```